### PR TITLE
[AssetMapper] Javascript sequence parser

### PR DESCRIPTION
**Pull request by smnandre**
| Q             | A
| ------------- | ---
| Branch?       | 7.2 for features
| Bug fix?      | yes
| New feature?  | yes/
| Deprecations? | yes
| Issues        | Fix #... 
| License       | MIT

Reopen of https://github.com/symfony/symfony/pull/54134 after https://github.com/symfony/symfony/pull/58944

### CODE REVIEW

**Comment by tacman** [comment]
I'm traveling for the Thanksgiving holiday, but I'll try to take a look.  Thanks.

File: src/Symfony/Component/AssetMapper/Compiler/JavaScriptImportPathCompiler.php [hunk 1]
```diff
@@ -27,9 +28,6 @@
  */
 final class JavaScriptImportPathCompiler implements AssetCompilerInterface
 {
-    /**
-     * @see https://regex101.com/r/1iBAIb/2
```
Comment (dunglas) [inline src/Symfony/Component/AssetMapper/Compiler/JavaScriptImportPathCompiler.php:31]: You should keep this comment

Comment (smnandre) [reply src/Symfony/Component/AssetMapper/Compiler/JavaScriptImportPathCompiler.php:31]: Missed this, thank you!

**Comment by smnandre** [comment]
Carson is not joking with the rules... sorry :(

**Pull request closed without merging**